backend/src/main/resources/db/migration/V1_0_11__Update_Information_About_Profile.sql

```diff
@@ -0,0 +1,29 @@
+ALTER TABLE credentials
+    ADD COLUMN phone_number NUMERIC(9,0) NULL;
```

Comment:
Model PhoneNumber value object and delegate validation to it

Reply:
Hello, the PhoneNumber we have do the validation in the Controller. Do you believe it is better?

Reply:
It's done